_po4a-tools/po4a-create-all-targets.sh

```diff
@@ -99,16 +99,17 @@ process_with_po4a () {
             THRESHOLD="$THRESH_VAL"
         fi
 
-        # Determine file format to be used
+        # Determine file format to be used and set options (configured as an array to allow adding an arbitrary number of them)
+        OPTION=()
         if [ $ext == yml ] ; then
             FILE_FORMAT=yaml
-            OPTION="skip_array"
+            OPTION=("skip_array")
         elif [[ $ext == html || "$filename" == *'-index' ]] ; then # '-index.md' has a markdown extension but is actually html and should be processed as such by po4a
             FILE_FORMAT=xml
-            OPTION="ontagerror=warn"
+            OPTION=("ontagerror=silent" "attributes=<img>src <img>alt")
```

Comment:
ontagerror=silent sounds like making debugging harder.

Reply:
Ok - I just remembered @softins made a change that got rid of the errors, so I'll leave it with `ontagerror=warn` then.